src/components/Search.tsx

```diff
@@ -21,10 +28,10 @@ type SearchBarProps = {
 
 const SearchBar = ( { handleTextChanges }: SearchBarProps ) => (
   <View style={styles.searchBar}>
-    <Icon name="magnify" size={25} />
+    <Icon name="magnify" size={25} style={styles.sharedCentering} />
```

Comment:
I would not call this `styles.sharedCentring`, but maybe `styles.centred`.

Even further, I would not apply utility styles directly, but instead keep the stlye names semantic, and then either repeat those properties, or have some utilities that are applied inside the semantic styles themselves. Can look at the latter aspect in #115